src/permitInfo/permitInfo.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$id": "https://cow.fi/schemas/token-list/v0.0.1.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "PermitInfo",
+  "description": "Permit info for tokens",
+  "type": "object",
+  "patternProperties": {
+    "^0x[a-fA-F0-9]{40}$": {
+      "oneOf": [
+        {
+          "type": "object",
+          "title": "PermitInfo",
+          "description": "Individual permit info when a token is known to be permittable",
+          "properties": {
+            "version": {
+              "type": "string",
+              "description": "Optional version number, as a string",
+              "pattern": "^\\d+$"
+            },
+            "type": {
+              "type": "string",
+              "description": "Type of permit",
+              "enum": [
+                "eip-2612",
+                "dai-like"
```

Comment:
nit: if we want a more homogeneous definition, we can add an option here for `unsupported` so we don't need the `oneOff` which can give problems or is harder interface.

 I should have said this in permit PRs, and also in the issue where you mentioned the schema. Anyways, is not too important, and yours is of course slightly more compact, so feel free to disregard the comment

Reply:
I like the suggestion, actually.
It's just that it creates some work for the CoW Swap implementation.
I'll create a task as a good to have and address it if there's time.

Reply:
https://github.com/cowprotocol/token-lists/issues/295

Reply:
yes, not an issue. thx for tracking the suggestion anyways